docs/manual/layout/templates/template-data.de.md

```diff
@@ -3,8 +3,7 @@ title: "Template-Daten anzeigen"
 description: "Alle Template-Daten anzeigen."
 url: "layout/templates/data"
 aliases:
-    - /de/templates/data/
```

Comment:
This deletes an alias. Was the article never linked on that alias?

Reply:
No, there wass only one file within /de/theme-manager/templates-verwalten/ before.